### ISSUE TITLE
Needs 2.x RC tag

### ISSUE THREAD

**Issue by elliot-sawyer** [Firesphere/silverstripe-subsite-solr#4]
Unable to use with solr-search (2.0.0-rc1) because this requires ^1.  Can you tag 2.0.0-rc1 to require solr-search ^2?

**Comment by Firesphere** [comment]
Yeah, I'll try to tag all sub-mobules and release them all at 2.0.0, and maybe try to remove the dependency on Guzzle while at it

**Comment by Firesphere** [comment]
Please check if you can get a working installation by making the solr-search version `dev-Firesphere-patch-1 as 2.0.0`?
